components/availability/src/Availability.svelte

```diff
@@ -826,6 +826,19 @@
       slots: [],
     });
   }
+
+  // Track and dispatch active dates whenever they change
+  let canonicalDates: Date[] = [];
+  $: if (
+    JSON.stringify(canonicalDates) !==
+    JSON.stringify(days.map((d) => d.timestamp))
```

Comment:
nit - Let's keep this consistent
```suggestion
    JSON.stringify(days.map((day) => day.timestamp))
```